src/algebra/big_operators/fin.lean

```diff
@@ -206,7 +206,33 @@ begin
 end
 
 end partial_prod
+section mul_nth
 
+variables [has_mul α] {n : ℕ}
+
+/-- Sends `(g₀, ..., gₙ)` to `(g₀, ..., gⱼgⱼ₊₁, ..., gₙ)`. -/
+@[to_additive "Sends `(g₀, ..., gₙ)` to `(g₀, ..., gⱼ + gⱼ₊₁, ..., gₙ)`."]
+def mul_nth (j : ℕ) (g : fin (n + 1) → α) (k : fin n) : α :=
```

Comment:
What do you think about changing this to
```suggestion
def contract_tuple (j : ℕ) (op : α → α →α) (g : fin (n + 1) → α) (k : fin n) : α :=
```
which gives you both the additive and multiplicative versions at once?

Reply:
This probably also belongs in data/fin/tuple rather than here

Reply:
Sure, sounds good to me; I didn't know about `insert_nth`